ClientKit/Display.cs

```diff
@@ -26,7 +26,9 @@
 using ViewerCount = System.UInt32;
 using EyeCount = System.Byte;
 using SurfaceCount = System.UInt32;
-using DistortionPriority = System.Int32;
+using DistortionPriority = System.Int32;
+using DisplayInputCount = System.Byte;

```

Comment:
am I correct in remembering that .NET doesn't distinguish signed vs. unsigned for int8s?